### DISCUSSION TITLE
Code differences between linux - windows 10

### DISCUSSION THREAD

**Discussion by demosthenesk** [XusinboyBekchanov/VisualFBEditor#112] [General]
The following code gives different results in linux and windows10.

```
		With This
			.Name = "Form1"
			.Text = "Form1"
			.StartPosition = FormStartPosition.CenterScreen
			.SetBounds 0, 0, 350, 300
		End With

```
In Windows 10 the form is centered to the screen
In Linux ubuntu-mate 20.04 SetBounds 0, 0 overwrites CenterScreen and form is running at x=0, y=0

**Comment by demosthenesk** [comment]
To center in screen in Linux i must write StartPosition after SetBounds

```
		With This
			.Name = "Form1"
			.Text = "Form1"
			.SetBounds 0, 0, 350, 300
			.StartPosition = FormStartPosition.CenterScreen
		End With

```

**Reply by XusinboyBekchanov** [reply]
Fixed: Form StartPosition property in Linux
https://github.com/XusinboyBekchanov/MyFbFramework/commit/aa3361b9f2366636d2561c8cf0d9954f91c4dc03

**Reply by XusinboyBekchanov** [reply]
We have opened this discussion well, we can accumulate all the differences here, if possible, we will correct it.

**Reply by demosthenesk** [reply]
ok!
